notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb

```diff
@@ -958,10 +889,10 @@
         "\n",
         "First, you define a component to import the evaluation metrics for the challenger model to the Model Registry. The component takes the following arguments:\n",
         "\n",
-        "- display_name: Human readable name for the evaluation metrics\n",
-        "- metrics: The evaluation metrics formatted for classification.\n",
-        "- parent_model_resource: The full resource name for the challenger model version.\n",
-        "- region: The region."
+        "- `display_name`: Human readable name for the evaluation metrics\n",
+        "- `metrics`: The evaluation metrics formatted for classification.\n",
+        "- `parent_model_resource`: The full resource name for the challenger model version.\n",
+        "- `region`: The region where you want to create/use the resources."
```

Comment:
remove / and change to:
- The region where you want to create or use the resources.